dev/com.ibm.ws.session.cache/src/com/ibm/ws/session/store/cache/CacheStoreService.java

```diff
@@ -86,7 +86,7 @@ protected void activate(ComponentContext context, Map<String, Object> props) {
             try {
                 uri = new URI(uriValue);
             } catch (URISyntaxException e) {
-                // TODO log error here
```

Comment:
Why would we not fail immediately when URI is not valid?  It seems wrong to continue on with a null URI.
Also, the error message ought to show the actual error so that the user can debug instead of just telling them something is wrong with the value.

Reply:
I'll update the message so that the error is output rather than the URI.  I was torn on whether to fail out if the URI is not valid since I would generally expect the provider would fail out if given an invalid URI, but I suppose it makes sense for us to do it before even getting to the provider.